drivers/platform/stm32/stm32_i2c.c

```diff
@@ -71,6 +72,7 @@ int32_t stm32_i2c_init(struct no_os_i2c_desc **desc,
 		goto error_1;
 	}
 
+	sinit = param->extra;
```

Comment:
I think you mean i2cinit, right?

Reply:
Apologies, the wrong file got checked in. I've updated the code now